ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/model/SchemaNodeTest.java

```diff
@@ -39,7 +49,12 @@ public void shouldRoundTrip() {
   }
 
   @Test
-  public void shouldRoundTripWithoutFeatures() {
-    ModelTester.assertRoundTrip(NO_SERDE_FEATURES);
+  public void shouldRoundNoKeyFormat() {
```

Comment:
nit:
```suggestion
  public void shouldRoundTripNoKeyFormat() {
```
and similarly for the other test below.

Reply:
I'll add to the follow up PR.

Reply:
see https://github.com/confluentinc/ksql/pull/6366